# since the first commit and at least up to commit 6257, FTE has always included
# opus.h using 
#
# #include "opus.h"
#
# The problem is that opus headers are commonly stored inside a directory called
# "opus" which means that the above will fail unless that directory is passed as
# an additional include path to the compiler. FTE makefile does that, but we 
# end up with a fixed path that will break on systems that store their includes 
# in different directories.

--- fteqw-code-r6257-trunk/engine/Makefile
+++ ferneu/engine/Makefile
@@ -991,21 +991,20 @@
 
 COMMONLIBFLAGS=
 COMMONLDDEPS=
 CLIENTLIBFLAGS=$(COMMONLIBFLAGS) $(LIBOPUS_STATIC) $(LIBSPEEX_STATIC) $(OGGVORBISFILE_STATIC)
 SERVERLIBFLAGS=$(COMMONLIBFLAGS)
 CLIENTLDDEPS=$(COMMONLDDEPS) $(LIBOPUS_LDFLAGS) $(LIBSPEEX_LDFLAGS) $(OGGVORBISLDFLAGS)
 SERVERLDDEPS=$(COMMONLDDEPS)
 ifeq (1,$(USE_OPUS))
     LIBOPUS_STATIC=-DOPUS_STATIC
     LIBOPUS_LDFLAGS=-lopus
-    ALL_CFLAGS+=-I/usr/include/opus
 endif
 ifeq (1,$(USE_SPEEX))
     LIBSPEEX_STATIC=-DSPEEX_STATIC
     LIBSPEEX_LDFLAGS=-lspeex -lspeexdsp
 endif
 
 ifeq (1,$(USE_VORBISFILE))
     OGGVORBISFILE_STATIC=-DLIBVORBISFILE_STATIC
 else
     OGGVORBISLDFLAGS=
--- fteqw-code-r6257-trunk/engine/client/snd_dma.c
+++ ferneu/engine/client/snd_dma.c
@@ -380,21 +380,21 @@
 
 	snd_capture_driver_t *cdriver;/*capture driver's functions*/
 	void *cdriverctx;	/*capture driver context*/
 } s_voip;
 
 #ifdef HAVE_OPUS
 #define OPUS_APPLICATION_VOIP				2048
 #define OPUS_SET_BITRATE_REQUEST			4002
 #define OPUS_RESET_STATE					4028
 #ifdef OPUS_STATIC
-#include "opus.h"
+#include <opus/opus.h>
 #define qopus_encoder_create	opus_encoder_create
 #define qopus_encoder_destroy	opus_encoder_destroy
 #define qopus_encoder_ctl		opus_encoder_ctl
 #define qopus_encode			opus_encode
 #define qopus_decoder_create	opus_decoder_create
 #define qopus_decoder_destroy	opus_decoder_destroy
 #define qopus_decoder_ctl		opus_decoder_ctl
 #define qopus_decode			opus_decode
 #else
 #define opus_int32 int

